server/protocols/protocols-imap4/pom.xml

```diff
@@ -131,6 +131,16 @@
             <groupId>io.netty</groupId>
             <artifactId>netty</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.1.72.Final</version>
```

Comment:
should we have a property for all netty dependency?

Reply:
This is a testing dependency and this will disapear with the netty4 migration...